Source/Diagnostics/ReducedDiags/FieldProbe.H

```diff
@@ -114,6 +114,16 @@ private:
     /** Check if the probe is in the simulation domain boundary
      */
     bool ProbeInDomain () const;
+
+    /**
+     * Simple utility function to normalize the components of a "vector"
+     */
+    void normalize(amrex::Real &x, amrex::Real &y, amrex::Real &z){
```

Comment:
```suggestion
    void normalize(amrex::Real AMREX_RESTRICT &x, amrex::Real AMREX_RESTRICT &y, amrex::Real AMREX_RESTRICT &z){
```

Reply:
Looks like some of the compilers are not happy with this. Do you recommend this change just to allow the compiler to better optimize this function or is there some further value to "restricting" the input? If it is just for optimization we could skip it since this function is only called during initialization (at present at least).

Reply:
@ax3l I pushed a change that keeps the restrict but will satisfy the builds that failed. I disabled the auto-merge to give you a chance to confirm this change is appropriate before merging.